pkg/adapter/client_unix.go

```diff
@@ -14,7 +14,14 @@ func formatDefaultBridge(remoteConn *remoteclientconfig.RemoteConnection, logLev
 	if port == 0 {
 		port = 22
 	}
+	options := ""
+	if remoteConn.IdentityFile != "" {
+		options += " -i " + remoteConn.IdentityFile
+	}
+	if remoteConn.IgnoreHosts {
+		options += " -q -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
```

Comment:
Out of curiosity: Why enabling the quite mode only here?

Reply:
Actually I didn’t have it anywhere, but got tired if the “WARNING adding host key” output on every single command